role-service/src/main/java/uk/ac/ox/ndph/mts/role_service/model/Role.java

```diff
@@ -0,0 +1,65 @@
+package uk.ac.ox.ndph.mts.role_service.model;
+
+import lombok.Getter;
+import lombok.Setter;
+import org.hibernate.envers.Audited;
+import org.springframework.data.annotation.CreatedBy;
+import org.springframework.data.annotation.CreatedDate;
+import org.springframework.data.annotation.LastModifiedBy;
+import org.springframework.data.annotation.LastModifiedDate;
+import org.springframework.data.jpa.domain.support.AuditingEntityListener;
+
+
+import javax.persistence.Entity;
+import javax.persistence.EntityListeners;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+import javax.persistence.GenerationType;
+import javax.persistence.SequenceGenerator;
+import javax.persistence.Column;
+import javax.validation.constraints.NotBlank;
+import java.time.LocalDateTime;
+
+@Entity
+@EntityListeners(AuditingEntityListener.class)
+@Audited
+public class Role {
+    @Id
+    @GeneratedValue(strategy = GenerationType.SEQUENCE, generator = "role_sequence")
+    @SequenceGenerator(name = "role_sequence", sequenceName = "role_sequence", allocationSize = 1)
```

Comment:
This means that we can have roles defined with a specific ID as described in the trial config example.  I don't have a specific ask here...

Reply:
Yes.  I wonder whether it makes more sense to reference them by (unique) role name in the json, instead of a generated int?